waggle-dance-core/src/test/java/com/hotels/bdp/waggledance/client/DefaultMetaStoreClientFactoryTest.java

```diff
@@ -101,7 +107,21 @@ public void defaultMethodCallThrowsTransportExceptionRetries() throws TException
 
     String result = iface.getName();
     assertThat(result, is("ourName"));
-    verify(base).reconnect();
+    verify(base).reconnect(HiveUgiArgs.WAGGLE_DANCE_DEFAULT);
+  }
+
+  @Test
+  public void set_ugi_before_call() throws Exception {
```

Comment:
Is this the only test related to set_ugi? If that's the case I would say we could also test when you set a custom ugi_value. We can verify the custom value get sets over the default one.

Reply:
this is a custom value